install/kubernetes/helm/istio/charts/certmanager/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+appVersion: latest
+description: A Helm chart for Kubernetes
```

Comment:
Please use a proper description ( I assume it's cut&pasted and we have the same in all...)

Reply:
I hope I did it right